### PR TITLE
tests: Implement multi operation Txn

### PR DESCRIPTION
**Pull request by serathius**
cc @ahrtr @ptabor

### CODE REVIEW

File: tests/linearizability/traffic.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ package linearizability
 import (
 	"context"
 	"fmt"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/tests/v3/linearizability/model"
```
Comment (logicalhan) [inline tests/linearizability/traffic.go:21]: nit: import groups

File: tests/linearizability/model/history.go [hunk 2]
```diff
@@ -175,11 +177,96 @@ func (h *AppendableHistory) AppendTxn(key, expectValue, newValue string, start,
...
+
+func toEtcdCondition(cmp clientv3.Cmp) (cond EtcdCondition) {
+	switch {
+	case cmp.Result == etcdserverpb.Compare_EQUAL && cmp.Target == etcdserverpb.Compare_VALUE:
+		cond.Key = string(cmp.KeyBytes())
+		cond.ExpectedValue = ToValueOrHash(string(cmp.ValueBytes()))
+	case cmp.Result == etcdserverpb.Compare_EQUAL && cmp.Target == etcdserverpb.Compare_CREATE:
+		cond.Key = string(cmp.KeyBytes())
+	default:
+		panic(fmt.Sprintf("Compare not supported, target: %q, result: %q", cmp.Target, cmp.Result))
```
[the remark this answers is not in the document]
Comment (serathius) [reply tests/linearizability/model/history.go:224]: This is not a unit test. Reason to panic in model is that any error here is a programmer error. Example client started using compare operator that is not supported by model. That needs to be quickly surfaced as whole test/model is useless and it's reports are meaningless until this is fixed.

[the remark this answers is not in the document]
Comment (logicalhan) [reply tests/linearizability/model/history.go:224]: Oh this is actually a test file. I didn't mean this is a unit test, I meant you could unit test this function. But this is already a test file.

File: tests/linearizability/traffic.go (hunk 1 above)
Comment (serathius) [reply tests/linearizability/traffic.go:21]: done

File: tests/linearizability/model/history.go [hunk 3]
```diff
@@ -15,6 +15,8 @@
 package model
 
 import (
+	"fmt"
+	"go.etcd.io/etcd/api/v3/etcdserverpb"
```
[the remark this answers is not in the document]
Comment (serathius) [reply tests/linearizability/model/history.go]: done

File: tests/linearizability/traffic.go (hunk 1 above)
Comment (ptabor) [reply tests/linearizability/traffic.go]: FYI: `./scripts/fix.sh` takes care of the import groups.

**Review by ptabor — APPROVED** [review]

**Pull request merged**